agave_app/Controls.h

```diff
@@ -191,22 +193,25 @@ private slots:
   QSlider m_slider;
 };
 
-class Controls
+class AgaveFormLayout : public QGridLayout

```

Comment:
introducing this class allowed finer control over laying out collections of input controls and lets us ensure some consistency across the ui.